src/Plugins/RpcServer/RpcException.cs

```diff
@@ -39,5 +40,17 @@ public static void ThrowIfNull<T>(T value, string paramName, RpcError error)
         {
             if (value is null) throw new RpcException(error.WithData($"Parameter '{paramName}' is null"));
         }
+
+        /// <summary>
+        /// Throws an exception if the number of parameters is less than the required number.
+        /// </summary>
+        /// <param name="params">The parameters to check.</param>
+        /// <param name="requiredArgs">The required number of parameters.</param>
+        /// <param name="error">The error code to throw.</param>
+        public static void ThrowIfTooFew(JArray @params, int requiredArgs, RpcError error)
+        {
+            if ((@params is null && requiredArgs > 0) || (@params is not null && @params.Count < requiredArgs))
+                throw new RpcException(error.WithData($"Too few arguments. Expected at least {requiredArgs} but got {@params?.Count ?? 0}."));
```

Comment:
i think this type of exception messgae is too general, would be better to specify which is missing for dedicated rpc methods. but better than nothing.